src/stemmabench/algorithms/manuscript_base.py

```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
+from typing import Union, Dict
+import os
+
+
```

Comment:
Rename the class into a more relevant name : I suggest ManuscritInTree or something like that, to show that it is at the stemma scale.

Reply:
Renamed to ManuscriptInTreeBase